scripts/functions.netconfig

```diff
@@ -48,7 +48,7 @@ find_current_service()
 {
     local name=$1
     local pfx file link
-    for pfx in /etc/systemd /var/run/systemd /run/systemd ; do
+    for pfx in /etc/systemd /run/systemd /run/systemd ; do
```

Comment:
```suggestion
    for pfx in /etc/systemd /run/systemd ; do
```

Reply:
right, it was to catch /var/run -> /run move ages ago, not needed any more. fixed.